moveit_core/robot_state/test/test_kinematic.cpp

```diff
@@ -318,15 +318,15 @@ TEST(FK, OneRobot)
       "    <limit effort=\"100.0\" lower=\"0.0\" upper=\"0.19\" velocity=\"0.2\"/>"
       "    <parent link=\"link_c\"/>"
       "    <child link=\"link_d\"/>"
-      "    <origin rpy=\" 0.0 0.1 0.0 \" xyz=\"0.1 0.1 0 \"/>"
+      "    <origin rpy=\" 0.0 0.0 0.0 \" xyz=\"0.1 0.1 0 \"/>"
```

Comment:
Keeping the orientation facilitates validation of link transforms for mimic joints, which wasn't done before).